pycalphad/tests/test_database.py

```diff
@@ -461,8 +461,8 @@ def test_comma_templims():
      TYPE-DEF % SEQ * !
      TYPE-DEF A GES AMEND_PHASE_DESCRIPTION @ MAGNETIC -3 0.28 !
      TYPE-DEF B GES AMEND_PHASE_DESCRIPTION @ MAGNETIC -1 0.4 !
-     TYPE-DEF O GES AMEND_PHASE_DESCRIPTION B2_BCC DIS_PART A2_BCC !
-     TYPE-DEF Y GES AMEND_PHASE_DESCRIPTION FCC_4SL DIS_PART A1_FCC !
+    $ TYPE-DEF O GES AMEND_PHASE_DESCRIPTION B2_BCC DIS_PART A2_BCC !
+    $ TYPE-DEF Y GES AMEND_PHASE_DESCRIPTION FCC_4SL DIS_PART A1_FCC !
```

Comment:
Why are we commenting these out again?

Reply:
The test is defined in and explicitly used for the `test_comma_templims` test. If the FCC_4SL type definition were defined, it would raise errors because this test database is truncated and does not define the FCC_4SL phase. I added an additional comment to clarify why the fcc type definition is commented.